### DISCUSSION TITLE
Input Data for ReceiverLowEarthOrbiter

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#77] [Q&A] [answered]
### Description

Hello
We are trying to create the *accuracyDefiniton,*receiverDefinition,... files for a cubesat constellation. I have looked through the config files for gracefo and sentinel1a and was trying to figure where the different sources where for the content of the 2 files ( *antennas.txt and * temporal.txt) with the idea of understanding the source information so that i can apply the same information to our low earth receivers. Or maybe if i dont have the information yet available then i need to try to get it

TXS for the help
Cheers
Jean

### Operating systems

- [ ] Linux
- [X] Windows
- [ ] macOS
- [ ] Other (please specify in the description)

**Comment by Bemtgenj** [comment]
Hello 
i like to add a couple more comments. The same question is valid for the satellite info data. Why is there a center of gravity file for Sentinel1A but not for example for Gracefo1 - or did i miss this file for gracefo1?
txs for the help and clarification

Cheers
Jean

**Comment by Bemtgenj** [comment]
Update 
some information like antenna x,y,z i fund in teh document " Sentinel-1 POD Properties for GPS POD, i will look if there is some more information thatwill be needed for the input fies

Cheers
Jean 

https://sentinel.esa.int/documents/247904/3455957/Sentinel-1-properties-for-GPS-POD

**Comment by Bemtgenj** [comment]
Hello

Update  - Was the Sentinel1 satellite technical information that you used  contained in the Sentinel1 Flight Operations Manual Vol 3 (as some other document references this one)? If yes do you know where i can find it? i have not been able to locate it on the web.

Tx for the help and the feedback

Cheers
Jean

**Comment by bsuesser-rechberger** [comment, marked as answer]
Hello Jean, 

the meta data for the antennas.txt file, we got from: Sentinel-1-properties-for-GPS-POD.pdf and from the antex file which you can find at https://sentinels.copernicus.eu/web/sentinel/technical-guides/sentinel-1-sar/pod/satellite-parameters.
Please note: During the POD done by ESA, they found out that the z-component value of the PCO (z = 97 mm) provided from
the manufacturer has to be replaced by  z = 68 mm, which produces more consistent results.

The meta data for the temporal.txt file we got from Sentinel-1-properties-for-GPS-POD.pdf.

Best regards,
Barbara

**Comment by Bemtgenj** [comment]
Hello Barbara

thank you very much for the information, i will take a look at the web site and the document you send me.

Cheers
jean
